rdrf/rdrf/patients_listing.py

```diff
@@ -455,6 +475,7 @@ def icon(self, tick):
 
 class ColumnWorkingGroups(Column):
     field = "working_groups_display"
+    sort_fields = ["working_groups_display"]
```

Comment:
I might be missing something but can't we just use `working_groups__name` here and drop all the special case translating code above?